pkg/auth/azure_test.go

```diff
@@ -51,11 +51,6 @@ func TestValidateTokenStatic(t *testing.T) {
 		ExpectedError error
 		noInit        bool
 	}{
-		{
-			Name:          "Not a token",
-			Token:         "asdf",
-			ExpectedError: errMatcher{"Failed to parse the JWT.\nError: token is malformed: token contains an invalid number of segments"},
```

Comment:
I don't quite understand why we need to delete this test. If we provide "asdf", surely the library will not accept that?
I would rather expect a more generic error here than just deleting the test.